### PR TITLE
VEN-181 | Set up dev dependencies

### PR DESCRIPTION
**Pull request by Nizar-Rahme**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+src/serviceWorker.ts
```
Comment (midori0507) [inline .eslintignore]: Do you want to snatch gitignore config from youth-ui or kukku ? Or in separate PR?

File: src/locales/en.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "harbours_list": {
```
Comment (midori0507) [inline src/locales/en.json]: should we camelCase ?

File: .eslintignore (hunk 1 above)
Comment (Nizar-Rahme) [reply .eslintignore]: I guess it's the same that CRA generates.

File: src/locales/en.json (hunk 2 above)
Comment (Nizar-Rahme) [reply src/locales/en.json]: I don't know, any conventions we agreed on following?

File: .eslintignore (hunk 1 above)
Comment (midori0507) [reply .eslintignore]: Sorry, wrong file, thought this is gitignore

File: src/locales/en.json (hunk 2 above)
Comment (midori0507) [reply src/locales/en.json]: Not sure if we have convention on this, been camelCase in translation all my life :D Just asking

**Pull request merged**